graylog2-web-interface/src/views/components/CreateNewWidgetModal.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { useMemo } from 'react';
+import styled, { css } from 'styled-components';
+
+import { Modal, Button } from 'components/bootstrap';
+import type WidgetPosition from 'views/logic/widgets/WidgetPosition';
+import usePluginEntities from 'hooks/usePluginEntities';
+import { WidgetActions } from 'views/stores/WidgetStore';
+import generateId from 'logic/generateId';
+import { CurrentViewStateActions } from 'views/stores/CurrentViewStateStore';
+import { useStore } from 'stores/connect';
+import { ViewStore } from 'views/stores/ViewStore';
+
+const modalTitle = 'Create new widget';
+
+const WidgetList = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+`;
```

Comment:
Let's add a `gap` here to ensure there is always a gap between each widget tile.

Reply:
I also like how it looks, if we would center the tiles.
<img width="609" alt="image" src="https://user-images.githubusercontent.com/46300478/210577207-347061a8-7a94-40f0-8355-2821bc7e6ac1.png">

In this example I also used `background: transparent` and `border-radius: 4px;` for the tiles.